Source/ASTableView.mm

```diff
@@ -96,6 +96,16 @@ - (void)didTransitionToState:(UITableViewCellStateMask)state
   [super didTransitionToState:state];
 }
 
+//- (BOOL)isAccessibilityElement
```

Comment:
Remove this code? Yeah, it does seem likely that we should be relying on UITableView's own implementations anyway.

Reply:
Done

Reply:
Done